services/database/user.ts

```diff
@@ -35,4 +35,17 @@ async function getUsers() {
     return users;
 }
 
-export { getUser, getUsers };
+/**
+ * Creates a new user corresponding to newUserData and returns the newly created user
+ * @param newUserData - data corresponding to the new user
+ * @returns - the newly created user
+ */
+async function createUser(newUserData) {
+    const user = await prisma.user.create({
```

Comment:
We should have a helper function validating the data before we attempt to create a record in the database.